lib/flowMain.js

```diff
@@ -29,27 +29,35 @@ const languages = [
 	'javascriptreact'
 ]
 
-export function activate(context:ExtensionContext): void {
+export function activate(context: ExtensionContext): void {
 	//User can disable flow for some projects that previously used flow, but it's not have actual typing
 	if (!isFlowEnabled()) {
 		return
 	}
-	global.vscode = vscode
 
-	setupLogging()
-	checkNode()
-	checkFlow()
+	findFlowConfigDir(vscode.workspace.rootPath).then(configPath => {
+		if (!configPath) {
+			console.log(`Not starting Flow: Could not find a .flowconfig after a backwards search from ${vscode.workspace.rootPath}.`)
```

Comment:
This ends up in the vscode developer tools web inspector, which is better than nothing IMO.